src/ZoweUSSNode.ts

```diff
@@ -282,7 +282,10 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
         this.tooltip = injectAdditionalDataToTooltip(this, newFullPath);
 
         vscode.commands.executeCommand("zowe.uss.refreshUSSInTree", this);
-        vscode.commands.executeCommand("zowe.uss.ZoweUSSNode.open", this);
+        if (this.contextValue === extension.DS_TEXT_FILE_CONTEXT ||
```

Comment:
I would remove those changes, because the same logic is introduced in fix for rename:
https://github.com/zowe/vscode-extension-for-zowe/pull/619

Reply:
@stepanzharychevbroadcom I've removed the USS folder rename changes from this PR to avoid merge conflicts when your changes from #619 come in.